src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,22 +2,29 @@
 
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
 
 public class SalaryInfo {
 
+    private static final int DATE_FROM_DATA = 0;
+    private static final int NAME_FROM_DATA = 1;
+    private static final int HOURS_FROM_DATA = 2;
+    private static final int SALARY_FROM_DATA = 3;
     private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
+    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
+            throws ArrayIndexOutOfBoundsException {
```

Comment:
a suggestion to handle the exception in the method rather than adding it to the method signature